exercises/06-exercise.py

```diff
@@ -0,0 +1,86 @@
+'''
+	Author: Jorge Diaz Jr
+
+    Exercise 6
+    Can you now turn your result from exercise 4 into an object?
+
+    The idea is to run:
+    myreader = ANReader(filename, chunk_size=1024)
+    # get the 10th chunk of 1024 rows as a pandas dataframe
+    row = myreader(10)
+
+'''
+import pandas as pd
+import os
+
+class ANREADER:
+	'''
+	resource_path and chunk size are arguments passed in
+   	returns a dataframe object where the df is a specified
+   	chunk of data
+	'''
+	def __init__(self, resource_path, chunk_size = 1024):
+		'''
+		adds the chunks of data to a list
+		'''
+		self.chunks_of_data = []
+		for chunk in pd.read_csv(resource_path, chunksize=chunk_size, header=None):
+			self.chunks_of_data.append(chunk)	
+		pass
+
+	def __call__(self, chunk_num):
+		'''
+		returns specified chunk number/index from list of all chunks created
+		'''
+		result = self.chunks_of_data[chunk_num]
+		return result
+
+	def get_chunks(self):
```

Comment:
another thing you can do is include the `__len__` function, replace `get_chunks` with `__len__`. This way you can do:
```py3
f = ANREADER(...)
len(f) # gives the number of chunks
```